Database/Stored Procedures/Add_or_Update/add_or_update_podium.sql

```diff
@@ -192,4 +196,28 @@ GO
 BEGIN TRAN
 EXEC sp_add_or_update_podium 666, 1, 3, 'Tent 1', 1200, 600, 600, 100, 12000, 12, 1000, 450, 450, 'Veel ruimte', 0
 ROLLBACK TRAN
+GO
+
+/* tent from wrong festival  */
+BEGIN TRAN
+EXEC sp_add_or_update_podium 1, 1, 2, 'Tent 1', 4000, 2000, 200, NULL, 999, 12, 1000, 1000, NULL, 'Bomen', 1
+ROLLBACK TRAN
+GO
+
+/* update tent from wrong festival */
+BEGIN TRAN 
+EXEC sp_add_or_update_podium 1, 1, 2, 'Tent 1', 4000, 2000, 200, NULL, 999, 12, 1000, 1000, NULL, 'Bomen', 0
+ROLLBACK TRAN
+GO
+
+
+-- Reviewed door Yuri
```

Comment:
Is het nodig deze tests ook nog hier op te slaan?

Reply:
Nee, deze tests testen niet eens de goede constraint.